libs/tools/barista/src/transform.ts

```diff
@@ -202,6 +203,51 @@ export const copyHeadlineTransformer: BaPageTransformer = async source => {
   return transformed;
 };
 
+/** Replaces absolute href with contentLink */
+export const relativeUrlTransformer: BaPageTransformer = async source => {
+  const transformed = { ...source };
+  if (source.content && source.content.length) {
+    transformed.content = runWithCheerio(source.content, $ => {
+      const links = $('a');
+      if (links.length) {
+        links.each((_, link) => {
+          const linkValue = $(link).attr('href');
+          if (linkValue && isRelativeUrl(linkValue)) {
+            let url = parse(linkValue);
+            // Link Value
+            if (url.pathname === null) {
+              $(link.attribs).append(
+                $(link)
+                  .removeAttr('href')
+                  .attr('contentLink', '/'),
+              );
+            } else {
+              $(link.attribs).append(
+                $(link)
+                  .removeAttr('href')
+                  .attr('contentLink', url.pathname),
+              );
+            }
```

Comment:
This could probably be simplified to keep the code a little bit more dry
```suggestion
              $(link.attribs).append(
                $(link)
                  .removeAttr('href')
                  .attr('contentLink', url.pathname || '/'),
              );
```